src/actions/receive-profile.js

```diff
@@ -47,10 +47,10 @@ import {
   initializeSelectedThreadIndex,
   initializeHiddenGlobalTracks,
   getVisibleThreads,
-} from '../profile-logic/tracks';
-import { computeActiveTabTracks } from '../profile-logic/active-tab';
-import { setDataSource } from './profile-view';
```

Comment:
I'm still seeing some imports that are happening in the `./` directory. Can you revert all of these ones like I suggested before?